stdlib/private/CMakeLists.txt

```diff
@@ -19,3 +19,9 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
   add_subdirectory(SwiftPrivateDarwinExtras)
   add_subdirectory(SwiftPrivatePthreadExtras)
 endif()
+
+if(CMAKE_SYSTEM_NAME STREQUAL "FreeBSD")
```

Comment:
Any reason to not do an 'or' and add this to the previous 'if' statement?  I expect these three libraries to be buildable on any unix-y system.